api/tests/Feature/CommunityInterestTest.php

```diff
@@ -359,68 +361,45 @@ public function testCommunityInterestsPaginatedRoles(): void
     public function testCommunityInterestsPaginatedAuthorizedToView(): void
     {
         CommunityInterest::truncate();
-        $communityInterestWithBothInterests = CommunityInterest::factory()->create([
-            'user_id' => User::factory(),
-            'community_id' => $this->communityId,
-            'job_interest' => true,
-            'training_interest' => true,
-        ]);
-        $communityInterestWithJobInterest = CommunityInterest::factory()->create([
+        $communityInterestWithConsent = CommunityInterest::factory()->create([
             'user_id' => User::factory(),
             'community_id' => $this->communityId,
-            'job_interest' => true,
-            'training_interest' => false,
+            'consent_to_share_profile' => true,
         ]);
-        $communityInterestWithTrainingInterest = CommunityInterest::factory()->create([
+        $communityInterestWithoutConsent = CommunityInterest::factory()->create([
             'user_id' => User::factory(),
             'community_id' => $this->communityId,
-            'job_interest' => false,
-            'training_interest' => true,
-        ]);
-        $communityInterestWithNoInterests = CommunityInterest::factory()->create([
-            'user_id' => User::factory(),
-            'community_id' => $this->communityId,
-            'job_interest' => false,
-            'training_interest' => false,
+            'consent_to_share_profile' => false,
         ]);
         $otherCommunityInterest = CommunityInterest::factory()->create([
             'user_id' => User::factory(),
             'community_id' => Community::factory(),
-            'job_interest' => false,
-            'training_interest' => false,
+            'consent_to_share_profile' => false,
```

Comment:
I think consent_to_share_profile should be true here since if it's false it's unauthorised anyway.

Reply:
Good point!  That's a better test than before.
[fix tests](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13146/commits/3e86c896839af4a785762183171b2a2b37852925)